### ISSUE TITLE
REPL completion not showing axpy!

### ISSUE THREAD

**Issue by antoine-levitt** [JuliaLang/julia#23550]
```
julia> Base.axpy!
axpy! (generic function with 9 methods)

julia> Base.ax<TAB>
```
and nothing happens. Is this expected?

**Comment by fredrikekre** [comment]
Yes, it is not exported from base. Use `Base.LinAlg.axpy!`

**Comment by antoine-levitt** [comment]
That seriously hurts discoverability. E.g. BLAS.<TAB> does not show axpy!, which I would assume means axpy isn't wrapped.

**Comment by fredrikekre** [comment]
```julia
julia> BLAS.axp<TAB>
axpby! axpy!
```

**Comment by antoine-levitt** [comment]
Not on 0.6, great if it's fixed on master!

**Comment by fredrikekre** [comment]
Seems to be fixed by #22374
